doc/releases/release-notes-3.7.rst

```diff
@@ -1,6 +1,28 @@
 :orphan:
 
-.. _zephyr_3.7:
+.. _zephyr_3.7.1:
+
+Zephyr 3.7.1
+############
+
+This is an LTS maintenance release with fixes.
+
+Issues fixed
+************
+
+These GitHub issues were addressed since the previous 3.7.0 tagged release:
+
+
+Mbed TLS
+********
+
+Mbed TLS was updated to version 3.6.1. The release notes can be found at:
+https://github.com/Mbed-TLS/mbedtls/releases/tag/mbedtls-3.6.1
+
+Mbed TLS 3.6 is an LTS release that will be supported
```

Comment:
Think the release team handles doing this? @aescolar

Reply:
From my point of view, it is ok to add this section in this PR

Reply:
@nashif ^^